### PR TITLE
featured tag removed

### PR DESCRIPTION
**Pull request by MattEspoz**
When a challenge is finished (no more tasks remain that are "created" or "skipped") the "featured" flag is removed.

It is removed from the detail page, and the ChallengeResults page

https://github.com/maproulette/maproulette3/issues/1948

### CODE REVIEW

File: src/components/ChallengePane/ChallengePane.js [hunk 1]
```diff
@@ -118,6 +118,7 @@ export class ChallengePane extends Component {
 
   render() {
     const showingArchived = this.props.history.location.search.includes("archived=true");
+    console.log(this.props);
```
Comment (jschwarz2030) [inline src/components/ChallengePane/ChallengePane.js:121]: please remove

**Comment by jschwarz2030** [comment]
so we're not changing the challenge's featured status in the backend?

**Comment by MattEspoz** [comment]
good point! I'm on it !

File: src/components/ChallengePane/ChallengePane.js (hunk 1 above)
Comment (MattEspoz) [reply src/components/ChallengePane/ChallengePane.js:121]: yup will remove

**Comment by jschwarz2030** [comment]
Associated issue was closed by the reporter

**Pull request closed without merging**